<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndMain</class>
 <widget class="QMainWindow" name="wndMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hunters</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/icons/main.ico</normaloff>:/icons/main.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="1" column="0">
     <widget class="QTextBrowser" name="tbLog">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuGame">
    <property name="title">
     <string>Gra</string>
    </property>
    <addaction name="actGameNew"/>
    <addaction name="separator"/>
    <addaction name="actGameQuit"/>
   </widget>
   <widget class="QMenu" name="mnuHelp">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actHelpAbout"/>
   </widget>
   <widget class="QMenu" name="mnuTurn">
    <property name="title">
     <string>Tura</string>
    </property>
    <addaction name="actTurnToggle"/>
    <addaction name="actTurnNext"/>
   </widget>
   <addaction name="mnuGame"/>
   <addaction name="mnuTurn"/>
   <addaction name="mnuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actGameNew"/>
   <addaction name="separator"/>
   <addaction name="actTurnToggle"/>
   <addaction name="actTurnNext"/>
   <addaction name="separator"/>
   <addaction name="actHelpAbout"/>
  </widget>
  <action name="actGameNew">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/glyphs/new.png</normaloff>:/glyphs/new.png</iconset>
   </property>
   <property name="text">
    <string>Nowa...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actGameQuit">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/glyphs/quit.png</normaloff>:/glyphs/quit.png</iconset>
   </property>
   <property name="text">
    <string>Wyjdź</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actHelpAbout">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/glyphs/about.png</normaloff>:/glyphs/about.png</iconset>
   </property>
   <property name="text">
    <string>O...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actTurnToggle">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/glyphs/pause.png</normaloff>:/glyphs/pause.png</iconset>
   </property>
   <property name="text">
    <string>Zatrzymaj</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actTurnNext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/glyphs/forward.png</normaloff>:/glyphs/forward.png</iconset>
   </property>
   <property name="text">
    <string>Następna</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
